questions.md

```diff
@@ -0,0 +1,6 @@
+# Few more questions on Pull request
+
+* Is it possible to accept a pull request into a new branch?
+* Is a pull request the same as git pull?
```

Comment:
Pull requests are different in the sense that `git pull` is a command that fetches code/date from a remote and then merges it into your local repository, while pull requests is more of a social coding feature that allows folks to collaborate before merging this code/data into your remote master